impeller/tessellator/tessellator.cc

```diff
@@ -113,8 +124,37 @@ Tessellator::Result Tessellator::Tessellate(
     auto vtx = points[index];
     callback(vtx);
   }
-
   return Result::kSuccess;
 }
 
+static void* heapAlloc(void* userData, unsigned int size) {
+  return malloc(size);
+}
+
+static void* heapRealloc(void* userData, void* ptr, unsigned int size) {
+  return realloc(ptr, size);
+}
+
+static void heapFree(void* userData, void* ptr) {
+  free(ptr);
+}
+
+static TESSalloc alloc = {
+    heapAlloc, heapRealloc, heapFree, 0, /* =userData */
+    16,                                  /* =meshEdgeBucketSize */
+    16,                                  /* =meshVertexBucketSize */
+    16,                                  /* =meshFaceBucketSize */
+    16,                                  /* =dictNodeBucketSize */
+    16,                                  /* =regionBucketSize */
+    0                                    /* =extraVertices */
```

Comment:
Consider adding a comment that these are in KB?

I think these are probably big enough for most cases, but also consider adding a unit test for a path that generates some pathological number of vertices to make sure it still works.

Reply:
These aren't KB from what I can tell, because the bucket size allocated is size * sizeof(WhatEverStruct). The default setting for meshFaceBucketSize is 512 and that generates a bucket size over 100KB

Reply:
I'll test on some pathological cases but I don't think its going to be an issue as the Tessellator will resize these as needed.

Reply:
Good call. I think it could be good to have a header with user defined literals for size units. Like the ones in chrono. In a later patch is fine.